src/Slider.jsx

```diff
@@ -270,7 +270,7 @@ class Slider extends React.Component {
 
     const points = Object.keys(marks).map(parseFloat);
     if (step !== null) {
-      const closestStep = Math.round(val / step) * step;
+      const closestStep = (Math.round((val - min) / step) * step) + min;
```

Comment:
Have we made a PR on the main repo with these changes?  Can you link here and take charge of making sure that's merged in when the main repo is updated?